python/cudf/cudf/tests/test_udf_rolling.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
```

Comment:
We should just move these tests to `test_rolling.py` - no need for a separate module